contracts/TicTacToe.sol

```diff
@@ -8,21 +8,22 @@ contract TicTacToe {
 		address player2;
 	}
 
-	Game[] public games;
+	uint public gameCounter;
```

Comment:
It seems like `gameCounter` is always just equal to the length of `games`. Can't we compute this with a `view` method instead of using a `storage` variable and save some gas?

Reply:
Yeah it is equal to the total number of games. The variable could be renamed to `totalGames`. 

`view` is only used for functions. The keyword `constant` is what you mean but we shouldn't use here because the variable increments within the `newGame` function.